StickKnightsOnline-Client/main.cpp

```diff
@@ -15,6 +15,7 @@
 #include <cstring>
 #include <fstream>
 #include <sstream>
+#include <math.h>
```

Comment:
Why is this needed for this change?

Also head's up, the naming convention for C++ versions of C libraries is `c"library_name"`

What I mean is, `<string.h>` is a C library and the equivalent C++ library is `<cstring>`

Reply:
So, `#include <cmath>` instead of `#include <math.h>` :)